### DISCUSSION TITLE
Roles connecting People/Orgs to Events

### DISCUSSION THREAD

**Discussion by saumier** [culturecreates/artsdata-data-model#129] [Controlled Vocabularies]
When capturing data from websites there is sometimes basic information about people/orgs related to the event. 

Schema.org has 2 commonly used properties: schema:organizer (recommended by Google) and schema:performer (recommended by Google). Schema:performer includes artists and participants on and off the stage.   In addition, cultural calendars sometimes have supporters (including government agencies) that can be important to highlight on webpages.

The basic trio of schema:organizer, schema:performer and schema:sponsor work well for most cases of scraping data published on websites.

However, in some cases there is more detailed information about the roles of people/orgs in an event.  

To capture these roles in depth, what should be the controlled vocabulary used by Artsdata?

**Comment by saumier** [comment]
Culture Creates worked on a first pass of roles that could be uses in a controlled vocabulary in 2022.  

This slide gives an overview of how the trio of organizer, performer and sponsor could be enriched with roles.
![Screenshot 2024-09-13 at 9 56 22 AM](https://github.com/user-attachments/assets/db98fe8c-2867-4cba-a3f3-122376e03cc6)

**Comment by saumier** [comment]
Geneviève Fabio, LaVitrine, shared the following roles with me via email.

> Rôles de contribution, voici la liste que nous utilisons et que nous avons amenée en V1. Nous savons pertinemment que cette liste va évoluer et nous sommes partis avec cette liste en provenance de Scène Pro et qui n'est pas exhaustive.
> Contribution de type Artiste
> Contribution de type Auteur
> Contribution de type Équipe de création
> Contribution de type Personnel technique
> Contribution de type Mise en scène
> Contribution de type Direction artistique
> Contribution de type Producteur
> Contribution de type Organisateur
> Contribution de type Promoteur
> Contribution de type Commanditaire
> Contribution de type Autre

![image](https://github.com/user-attachments/assets/660f838a-a5b3-418a-a56b-f6480585e7b0)

**Comment by fjjulien** [comment]
Je crois moi aussi qu'un vocabulaire contrôlé de rôles serait utile. 

En fait, je crois qu'il faudrait _des_ vocabulaires contrôlés.

Certains des rôles énumérés plus haut sont associés à un événement :

- diffuseur, promoteur, organisateur (ce sont des synonymes désignant le même rôle, c'est-à-dire l'entité (l'agent) qui détient ou a acquis les droits de vendre des billets pour la présentation d'un spectacle dans le cadre d'une ou de plusieurs représentations);
- co-diffuseur
- commanditaire (entité ayant fourni une contribution financière sous forme de commandite corporative et exigeant une contrepartie sous forme de visibilité)
- bailleur de fonds (entité ayant fourni un financement de fondation ou de source publique et n'exigeant pas de visibilité au même titre qu'un commanditaire.
  - Note: Pour une même représentation, il peut parfois y avoir des commanditaires ou des bailleurs de fonds soutenant soit la représentation elle-même, l'organisateur, la saison d'événements diffusés par l'organisateur, la tournée, ou la création du spectacle. De ces différent cas, la commandite de la représentation elle-même est le cas le moins fréquent. Le seul cas courant où un commanditaire est associé à un événement c'est le festival.

Certains rôles sont associés à un spectacle et impliquent un droit d'auteur par rapport au spectacle dans son ensemble ou à ses différents éléments créatifs ou de design : 

- Compagnie de production
- Direction artistique (équivalent de créateur, rôle qui échoit souvent de facto à la personne responsable de la mise en scène ou de la chorégraphie)
- Mise en scène
- Chorégraphie
  - La contribution de chorégraphe est parfois considérée comme un rôle de création associé à une oeuvre chorégraphique distincte du spectacle auquel la chorégraphie est intégrée)
- Différents rôles de conception
- Différents rôles technique (par exemple, régie)
- Autres rôles de soutien (par exemple, assistance à la mise en scène)

Certains rôles sont associés à une oeuvre dramatique ou musico-dramatique intégrée au spectacle (via la propriété [lrmoo:R74 uses expression of ](https://ontome.net/property/2238/namespace/218#summary)) : 

- Créateur/créatrice et ses sous-classes :
  - Auteur/autrice
  - Compositeur/compositrice
- Traduction
- Adaptation
- Rôles de soutien à la création (par exemple, conseiller dramaturgique)

Enfin certains rôles sont associés à l'exécution du spectacle sur scène mais sont déterminés par la compagnie de direction (et sont donc parfois considérés comme associés au spectacle plutôt qu'à la représentation) : 

- Interprète et ses sous-classes : 
  - acteur/comédien, marionnettiste, danseur, musicien, acrobate, etc.
- Interprète principal (par exemple, premier danseur, soliste)
- Direction musicale (incluant, par exemple, chef d'orchestre ou chef de choeur)

Bref, avant d'élaborer ces vocabulaires contrôlés, il faudrait commencer par définir et déclarer formellement la [classe Spectacle / Show](https://github.com/culturecreates/artsdata-data-model/discussions/133).

**Comment by fjjulien** [comment]
D'ici à ce que la communauté des arts de la scène ait établit un consensus sur les principales classes et sur les vocabulaires de rôles associées à chacune, il m'apparaît utile de proposer des lignes directrices décrivant et balisant les usages les plus courants de la propriété `performer`. 

Tel que mentionné dans [cette discussion](https://github.com/culturecreates/artsdata-data-model/discussions/142), on trouve beauboup de sources de données qui renseignent une valeur unique pour la propriété `performer`. Dans les cas les plus courants, l'entité désignée comme `performer` pourrait être considérée comme l'_artiste principal sur scène_. Dans d'autres cas fréquents, l'entité désignée comme `performer` est l'_entité principale responsable de la direction artistique_ (et, par extension, de la sélection des interprètes), c'est-à-dire la compagnie de production (comme c'est souvent le cas en danse ou en cirque).

À moins que je ne me trompe, `artiste principal sur scène` est une propriété de la classe `Offre de spectacle` dans Scène Pro. Si nous pouvions valider cette information et confirmer que c'est aussi le sens des relations Artiste-Spectacle dans Arts Touring Connector, cela constituerait un bon rôle par défaut pour les cas où il y a une seule entité renseignée sous la propriété `performer`.
